melodic/distribution.yaml

```diff
@@ -641,6 +641,21 @@ repositories:
       url: https://github.com/ros/cmake_modules.git
       version: 0.4-devel
     status: maintained
+  cnn_bridge:
+    doc:
+      type: git
+      url: https://github.com/wew84/cnn_bridge-doc.git
```

Comment:
While our infrastructure *can* handle a separate repository for documentation, it is much more common to just point the `doc` section here to the source repository (https://github.com/wew84/cnn_bridge.git).  I'd highly suggest that here given that the doc repository above is an empty repository.

Reply:
Accepted your comment and changed the doc repo